src/pytest_split/ipynb_compatibility.py

```diff
@@ -0,0 +1,60 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import List
+
+    from _pytest import nodes
+
+    from pytest_split.algorithms import TestGroup
+
+
+def _reorganize_broken_up_ipynbs(group: "TestGroup", items: "List[nodes.Item]") -> None:
```

Comment:
Now that this is in a dedicated module, this should be public, i.e. remove the `_` prefix 🙂

Also, I feel that the function name could be perhaps a bit more generic as the caller doesn't really need to know what it's doing under the hood. Would you mind changing it to e.g. `ensure_ipynb_compatibility`.

Reply:
Absolutely :) just pushed a fix.